lib/vimeo.js

```diff
@@ -417,13 +417,14 @@ Vimeo.prototype.generateClientCredentials = function (scope, fn) {
  * @param {Function}  errorCallback     Callback to be executed when the upload returns an error.
  */
 Vimeo.prototype.upload = function (
-  filePath,
+  file,
   params,
   completeCallback,
   progressCallback,
   errorCallback
 ) {
   var _self = this
+  var fileSize = file.size
```

Comment:
If `file` isn't an object, I think that this'll throw undefined property errors?

```suggestion
  var fileSize
```